BookingQueueSubscriber/BookingQueueSubscriber.Services/NotificationApi/AddNotificationRequestMapper.cs

```diff
@@ -212,7 +213,26 @@ public static AddNotificationRequest MapToNewHearingNotification(HearingDto hear
             }
             else
             {
-                notificationType = NotificationType.HearingConfirmationLipMultiDay;
+                if (usePostMay2023Template)
```

Comment:
Could you introduce a private method here to reduce the number of nested if statements? It would help make the mapper more readable

Reply:
good idea... done